### PR TITLE
Add IPAdapter benchmark and fix several bugs

### PR DESCRIPTION
**Pull request by zhtmike**
1. fix the bug when num_trials > 1 in IPAdapter inference code
2. fix the training optim for sd1.5 when group strategy is none

### CODE REVIEW

File: examples/ip_adapter/README.md [hunk 1]
```diff
@@ -345,6 +345,34 @@ python tools/prepare_coco.py --label path_of_captions_train2017.json --image pat
...
+|----------|---------------|------------------|-----------|-------|------------|------------|----------------|-------------|
+| 1.5      | D910*x1-MS2.2 | Image Variation  | DDIM      | 50    | 512x512    | 4          | 4.97           | 0.36        |
+| 1.5      | D910*x1-MS2.2 | Image-To-Image   | DDIM      | 30    | 512x512    | 4          | 4.81           | 0.52        |
+| 1.5      | D910*x1-MS2.2 | Image Inpainting | DDIM      | 35    | 512x768    | 4          | 2.69           | 0.26        |
+| 1.5      | D910*x1-MS2.2 | ControlNet       | DDIM      | 50    | 512x512    | 4          | 2.87           | 0.22        |
+| XL       | D910*x1-MS2.2 | Image Variation  | Euler EDM | 30    | 1024x1024  | 4          | 1.24           | 0.15        |
+| XL       | D910*x1-MS2.2 | ControlNet       | Euler EDM | 30    | 1024x1024  | 3          | 2.55           | 0.25        |
+
+> Context: {Ascend chip}-{number of NPUs}-{mindspore version}.
+> Speed (ste/s): sampling speed measured in the number of sampling steps per second.
```
Comment (hadipash) [inline examples/ip_adapter/README.md:373]: step/s

File: examples/stable_diffusion_v2/ldm/modules/train/optim.py [hunk 2]
```diff
@@ -43,14 +43,17 @@ def build_optimizer(
         _logger.info("Applying `%s` strategy for weight decay.", group_strategy)
 
     def decay_filter(param):
-        if group_strategy is not None and group_strategy.lower() == "unclip":
+        if group_strategy is None:
+            filter_list = ["layernorm", "bias"]
+        elif group_strategy.lower() == "unclip":
```
Comment (hadipash) [inline examples/stable_diffusion_v2/ldm/modules/train/optim.py:48]: Do we need to update `mindone/trainers/optim.py` as well?

File: examples/ip_adapter/README.md (hunk 1 above)
Comment (hadipash) [inline examples/ip_adapter/README.md:373]: step/s

**Review by hadipash — APPROVED** [review]

File: examples/ip_adapter/README.md (hunk 1 above)
Comment (zhtmike) [reply examples/ip_adapter/README.md]: Fixed, thx

File: examples/stable_diffusion_v2/ldm/modules/train/optim.py (hunk 2 above)
Comment (zhtmike) [reply examples/stable_diffusion_v2/ldm/modules/train/optim.py:48]: Updated. Thx for remind

**Review by Songyuanwei — APPROVED** [review]

**Review by SamitHuang — APPROVED** [review]

**Pull request merged**